include/k4a/k4a.hpp

```diff
@@ -301,13 +301,22 @@ class image
         return k4a_image_get_stride_bytes(m_handle);
     }
 
-    /** Get the image timestamp in microseconds
+    /** Get the image's device timestamp in microseconds
      *
-     * \sa k4a_image_get_timestamp_usec
+     * \sa k4a_image_get_device_timestamp_usec
      */
-    std::chrono::microseconds get_timestamp() const noexcept
```

Comment:
>std::chrono::microseconds get_timestamp() const noexcept [](start = 4, length = 56)

If this change makes it in for 1.1, we don't need to preserve this (this wrapper wasn't in 1.0), but if we're punting to 1.2, deleting this is a breaking API change - we may need to keep it for backwards compat and mark it deprecated :(

Reply:
Good call, I am going to see if we can fix this API in our 1.1 release 

---
In reply to: [284936947](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/pull/350#discussion_r284936947) [](ancestors = 284936947)